source/worksheet/page_setup.cpp

```diff
@@ -108,6 +112,10 @@ void page_setup::scale(double scale)
 
 double page_setup::scale() const
 {
+    if (!scale_.is_set())
```

Comment:
Alternative is to store a plain scale value (i.e. no optional) and initialise it with 100. Should we make the difference between unset and explicitly set to the default value.

Reply:
> Not using xlnt::optional means that we would have to be careful when handling (and not forget to handle) the default value of 100.
 
Not sure what you mean by 'not forget to handle', as it seams to me that the case `scale == 100` should always be handled correctly independently of whether we use an optional or not.

I also wonder if removing xlnt::optional for such use cases, would decrease our memory footprint (significantly), but that's for another PR/issue. So, feel free to leave it as is for now. We can discuss it later if needed.